esp/src/src-react/hooks/confirm.tsx

```diff
@@ -22,8 +23,9 @@ export function useConfirm({ title, message, onSubmit }: useConfirmProps): [Reac
             maxWidth={500}
         >
             <div>
-                {message.split("\n").map((str, idx) => {
-                    return <span key={idx}>{str} <br /></span>;
+                {message}<br /><br />
```

Comment:
@jeclrsg - should the `<br /><br />` be conditional on items.length?

Reply:
Yes, should have just wrapped the message in a paragraph to take care of that spacing.